JuliaGPU/session2/Feyaz/qn3.jl

```diff
@@ -0,0 +1,64 @@
+#qn3
+using BenchmarkTools
+
+function rootinator(num)
+	start=0
+	fin=num
+	ans=0
+	#step=0
+	while start<=fin
+		mid= trunc(Int, (start+fin)/2)
+		if(mid*mid==num)
+			ans=mid
+			break
+		end
+		if (mid*mid<num)
+			start=mid+1
+			ans=mid
+		else
+			fin=mid-1
+		end
+		#println(step)
+		#step=step+1
+	end
+	inc=0.1
+	prec=15			#for Int64, ig
+	#step=0
+	for i in 1:prec
+		while(ans*ans<=num)
+			ans=ans+inc
+		end
+		ans=ans-inc
+		inc=inc/10
+		#println(step)
+		#step=step+1
+	end
+	return(ans)
+end
+
+array_sqrt(arr)=	begin
```

Comment:
Could you define array_sqrt with just one line of code? Similar to how you found out sqrt of array using Julia's sqrt